Include/internal/pycore_bytesobject.h

```diff
@@ -33,6 +33,19 @@ _PyBytes_ReverseFind(const char *haystack, Py_ssize_t len_haystack,
                      const char *needle, Py_ssize_t len_needle,
                      Py_ssize_t offset);
 
+
+/** Helper function to implement the repeat and inplace repeat methods on a buffer
+ *
+ * len_dest is assumed to be an integer multiple of len_src.
+ * If src equals dest, then assume the operation is inplace.
+ *
+ * This method repeately doubles the number of bytes copied to reduce
+ * the number of invocations of memcpy.
+ */
+PyAPI_FUNC(void)
```

Comment:
It should not be exported.

Reply:
Shouldn't it? I think that's the way to be able to use it from the dynamically-loaded extension module created by `arraymodule.c`.

Reply:
Ya, I thought that `array` module was a builtin core module.